src/utils/crypto/index.ts

```diff
@@ -200,7 +202,12 @@ export async function encryptWithRecoveryKey(key: string) {
 export default CryptoWorker;
 
 export async function getFileHash(file: File | ElectronFile) {
-    const stream = await file.stream();
-    const hash = await cryptoGenericHash(stream);
+    let filedata: DataStream;
+    if (file instanceof File) {
+        filedata = getFileStream(new FileReader(), file, ENCRYPTION_CHUNK_SIZE);
```

Comment:
so using `ENCRYPTION_CHUNK_SIZE` feels weird when we  are not dealing with encryption, so have  `FILE_READER_CHUNK_SIZE` when referring to the chunk generated by the file reader, internally it's the same